app/assets/javascripts/Components/markus_file_manager.jsx

```diff
@@ -401,6 +402,7 @@ FileManager.defaultProps = {
     FolderOpen: <i className="fa fa-folder-open-o" aria-hidden="true" />,
     Delete: <i className="fa fa-trash-o" aria-hidden="true" />,
     Loading: <i className="fa fa-circle-o-notch fa-spin" aria-hidden="true" />,
+    Download: <i className="fa fa-download-o" aria-hidden="true" />,
```

Comment:
I don't think adding this is necessary (I think these props are just for the React component library we're using, not our own custom actions/icons).